src/timeout.ts

```diff
@@ -150,6 +150,11 @@ function isCSOTTimeoutContextOptions(v: unknown): v is CSOTTimeoutContextOptions
 
 /** @internal */
 export abstract class TimeoutContext {
+  start: number;
+  constructor() {
+    this.start = Math.trunc(performance.now());
```

Comment:
any reason you're not using our `now()` helper?

Reply:
We are already using `performance.now()` in the rest of this file, I kept up with the pattern.